repos/fdbt-site/tests/pages/api/defineCaps.test.ts

```diff
@@ -0,0 +1,86 @@
+import { CAPS_DEFINITION_ATTRIBUTE, FULL_CAPS_ATTRIBUTE, OPERATOR_ATTRIBUTE } from '../../../src/constants/attributes';
+import * as auroradb from '../../../src/data/auroradb';
+import defineCaps from '../../../src/pages/api/defineCaps';
+import * as sessions from '../../../src/utils/sessions';
+import { getMockRequestAndResponse, mockIdTokenMultiple } from '../../testData/mockData';
+import * as userData from '../../../src/utils/apiUtils/userData';
+import { ExpiryUnit } from '../../../src/interfaces/matchingJsonTypes';
+
+describe('defineCaps', () => {
+    const writeHeadMock = jest.fn();
+    const s3Spy = jest.spyOn(userData, 'putUserDataInProductsBucketWithFilePath');
+    s3Spy.mockImplementation(() => Promise.resolve('pathToFile'));
+
+    beforeEach(() => {
```

Comment:
We need some tests for the logic we have added to the ticketConfirmation.ts API - in the ticketConfirmation.test.ts file (create it if not there).